code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -2,100 +2,175 @@
 			new /obj/item/clothing/suit/hooded/chaplain_hoodie(src)
+			new /obj/item/weapon/storage/box/syndie_kit/imp_mindslave(src)
 			new /obj/item/weapon/card/id/syndicate(src)
+			new /obj/item/weapon/spellbook/oneuse/summonitem(src)
 			return
 
-		if("psycho")
-			new /obj/item/clothing/suit/raincoat(src)
-			new /obj/item/weapon/newspaper(src)
-			new /obj/item/weapon/twohanded/fireaxe(src)
-			new /obj/item/weapon/storage/fancy/cigarettes/cigpack_syndicate(src)
-			new /obj/item/clothing/shoes/sneakers/syndigaloshes(src)
-			new /obj/item/weapon/soap/syndie(src)
+		if("ninja")
+			new /obj/item/weapon/melee/energy/sword/saber(src)
+			new /obj/item/weapon/storage/belt/ninja/full(src)
+			new /obj/item/weapon/storage/box/throwing_stars(src)
+			new /obj/item/weapon/storage/box/throwing_stars(src)
+			new /obj/item/clothing/gloves/pickpocket(src)
+			new /obj/item/clothing/mask/balaclava(src)
+			new /obj/item/weapon/dnainjector/stealthmut/ninjabundle(src)
+			new /obj/item/weapon/spellbook/oneuse/smoke(src)
+			new /obj/item/weapon/spellbook/oneuse/summonitem(src)
 			return
 
 		if("boom")
-			new /obj/item/weapon/grenade/syndieminibomb(src)
-			new /obj/item/weapon/grenade/syndieminibomb(src)
-			new /obj/item/device/sbeacondrop/bomb(src)
-			new /obj/item/device/sbeacondrop/bomb(src)
-			new /obj/item/weapon/storage/box/syndie_kit/imp_microbomb(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
+			new /obj/item/weapon/c4(src)
```

Comment:
No other item containers use loops, and from what I've tested, they don't work for whatever reason unless I'm extremely dumb.

Which I probably am.